option_test.go

```diff
@@ -104,7 +108,7 @@ func TestOptionMarshalUnmarshal(t *testing.T) {
 						t.Fatalf("failed to unmarshal options: %v", err)
 					}
 
-					if diff := cmp.Diff(st.os, got, cmp.Comparer(addrEqual)); diff != "" {
+					if diff := cmp.Diff(st.os, got, cmp.Comparer(addrEqual), cmp.AllowUnexported(netip.Prefix{})); diff != "" {
```

Comment:
I'm sure there is a better way to get around this, not too familiar with go-cmp. Any recommendations?